designsafe/apps/workspace/static/designsafe/apps/workspace/scripts/services/apps-service.js

```diff
@@ -45,6 +45,30 @@
       });
     };
 
+    service.getSystemStatus = function(system) {
```

Comment:
Service looks good. Would it make more sense to move it to the Systems service instead of the Apps service?

Also it would be good to add a docstring with an example of what was returned from these calls.